roles/marathon_chronos/defaults/main.yml

```diff
@@ -0,0 +1,51 @@
+---
+chronos_image: "mesosphere/chronos"
+chronos_image_tag: "chronos-2.4.0-0.1.20150828104228.ubuntu1404-mesos-0.23.0-1.0.ubuntu1404"
+
+chronos_master: "zk://zookeeper.service.consul:2181/mesos"
+chronos_zk_hosts: "zookeeper.service.consul:2181"
+chronos_http_port: 14400
+chronos_mesos_framework_name: chronos
+chronos_cluster_name: mantl
+
+chronos_mesos_task_cpu: 0.1
+chronos_mesos_task_disk: 256
+chronos_mesos_zk_timeout: 10000
+
+#extra options
+chronos.assets_path:
```

Comment:
Do we *really* need such a list of empty variables?
I guess, those can be removed by now or, at least, commented out here and in `roles/marathon_chronos/templates/chronos-marathon.j2`.

@stevendborrelli what do you think?